shared/source/os_interface/driver_info.h

```diff
@@ -7,23 +7,54 @@
 
 #pragma once
 
+#include <cstdint>
 #include <string>
 
 namespace NEO {
 
 struct HardwareInfo;
 class OSInterface;
 
+struct PhysicalDevicePciBusInfo {
+    PhysicalDevicePciBusInfo(uint32_t domain, uint32_t bus, uint32_t device, uint32_t function)
+        : pciDomain(domain), pciBus(bus), pciDevice(device), pciFunction(function) {}
+
+    PhysicalDevicePciBusInfo(uint32_t bus, uint32_t device, uint32_t function)
+        : pciDomain(0), pciBus(bus), pciDevice(device), pciFunction(function) {}
+
+    PhysicalDevicePciBusInfo()
+        : PhysicalDevicePciBusInfo(InvalidValue, InvalidValue, InvalidValue, InvalidValue) {}
+
```

Comment:
why 3 seperate constructors?
Can we have 1?

Reply:
Additional constructors have been removed.